$MirOS: ports/devel/autoconf/2.61/patches/patch-lib_autoconf_general_m4,v 1.2 2008/04/05 17:59:49 tg Exp $
--- lib/autoconf/general.m4.orig	Sat Apr  5 23:04:25 2008
+++ lib/autoconf/general.m4	Sat Nov 21 23:28:30 2009
@@ -292,7 +292,7 @@ m4_define([AC_REVISION],
 # Autoconf, which is certainly not what the user intended.
 AU_DEFUN([AC_PREREQ],
 [m4_version_prereq([$1])[]dnl
-[AC_PREREQ(]]m4_defn([m4_PACKAGE_VERSION])[[)]])
+[AC_PREREQ(]]m4_defn([m4_PACKAGE_VERSIONNR])[[)]])
 
 
 # AC_PREREQ(VERSION)
@@ -304,7 +304,7 @@ m4_copy([m4_version_prereq], [AC_PREREQ]
 # AC_AUTOCONF_VERSION
 # -------------------
 # The current version of Autoconf parsing this file.
-m4_copy([m4_PACKAGE_VERSION], [AC_AUTOCONF_VERSION])
+m4_copy([m4_PACKAGE_VERSIONNR], [AC_AUTOCONF_VERSION])
 
 
 
@@ -1648,7 +1648,7 @@ AC_DEFUN([AC_CONFIG_AUX_DIR_DEFAULT],
 # do not automatically need to distribute the other auxiliary files.
 AC_DEFUN([AC_CONFIG_AUX_DIRS],
 [ac_aux_dir=
-for ac_dir in $1; do
+for ac_dir in $GNUSYSTEM_AUX_DIR $1; do
   if test -f "$ac_dir/install-sh"; then
     ac_aux_dir=$ac_dir
     ac_install_sh="$ac_aux_dir/install-sh -c"
